webapp/src/js/services/uhc-settings.js

```diff
@@ -0,0 +1,39 @@
+angular.module('inboxServices').factory('UHCSettings',
+  function() {
+    'use strict';
+    'ngInject';
+
+    const getMonthStartDate = settings => {
```

Comment:
Consider using the `Concise Body` syntax for lambdas here. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Function_body

Reply:
I don't like how the formatting looks if I do.